components/Cards/examples/CollectionViewController.m

```diff
@@ -0,0 +1,144 @@
+/*
+ Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import "CollectionViewController.h"
+#import "MaterialInk.h"
+#import "UICollectionViewController+MDCCardReordering.h"
+
+@interface CollectionViewController ()
+
+@end
+
+@implementation CollectionViewController
+
+static NSString *const kReusableIdentifierItem = @"itemCellIdentifier";
+@synthesize collectionViewLayout = _collectionViewLayout;
+
+- (instancetype)init {
+  UICollectionViewFlowLayout *defaultLayout = [[UICollectionViewFlowLayout alloc] init];
+  defaultLayout.minimumInteritemSpacing = 0;
+  defaultLayout.minimumLineSpacing = 1;
+  defaultLayout.scrollDirection = UICollectionViewScrollDirectionVertical;
+  return [self initWithCollectionViewLayout:defaultLayout];
+}
+
+- (instancetype)initWithCollectionViewLayout:(UICollectionViewLayout *)layout {
+  self = [super initWithCollectionViewLayout:layout];
+  if (self) {
+    _collectionViewLayout = layout;
+  }
+  return self;
+}
+
+- (void)viewDidLoad {
+  [super viewDidLoad];
+
+  [self.collectionView setCollectionViewLayout:self.collectionViewLayout];
+  self.collectionView.backgroundColor = [UIColor colorWithWhite:0.9 alpha:1];
```

Comment:
This will throw a warning on float conversion.
= [UIColor colorWithWhite:(CGFloat)0.9 alpha:1]